# HG changeset patch
# User Thomas Weber <thomas.weber.mail at gmail.com>
# Date 1237318561 -3600
#      Tue Mar 17 20:36:01 2009 +0100
# Node ID 75c502937d2cd4a25e0d6e703ef23da664aed77b
# Parent  a653c61ee98c4f3ac8e1fffef1382201074cafa6
Fix typo for termios.h
Thanks to Petr Salinger <Petr.Salinger at seznam.cz>

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2009-03-17  Thomas Weber  <thomas.weber.mail at gmail.com>
+
+        * configure.in: fix typo with regards to termios.h
+
 2009-05-25  Jaroslav Hajek  <highegg@gmail.com>
 
 	* examples/@FIRfilter/Makefile.in: New file.
 	* examples/Makefile.in: Include it.
 	* configure.in: Ditto.
 
 2009-05-24  Benjamin Lindner <lindnerb@users.sourceforge.net>
 
diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -1532,17 +1532,17 @@ AC_CHECK_HEADERS(assert.h curses.h direc
 
 ### C++ headers
 
 AC_LANG_PUSH(C++)
 AC_CHECK_HEADERS(sstream)
 AC_LANG_POP(C++)
 
 have_termios_h=no
-AC_CHECK_HEADERS($TERMIOS_H, have_termios_h=yes)
+AC_CHECK_HEADERS(termios.h, have_termios_h=yes)
 AC_CHECK_HEADERS(termio.h, have_termio_h=yes, have_termio_h=no)
 AC_CHECK_HEADERS(sgtty.h, have_sgtty_h=yes, have_sgtty_h=no)
 AC_CHECK_HEADERS(glob.h, have_glob_h=yes, have_glob_h=no)
 AC_CHECK_HEADERS(fnmatch.h, have_fnmatch_h=yes, have_fnmatch_h=no)
 AC_CHECK_HEADERS(conio.h, have_conio_h=yes, have_conio_h=no)
 
 ### I'm told that termios.h is broken on NeXT systems.
 
